osbs/build/build_request.py

```diff
@@ -99,6 +99,7 @@ def set_params(self, **kwargs):
         :param auto_build_node_selector: dict, a nodeselector for auto builds
         :param isolated_build_node_selector: dict, a nodeselector for isolated builds
         :param is_auto: bool, indicates if build is auto build
+        :param allowed_platform: str, indicates which platform should upload operator manifests
```

Comment:
is this a valid parameter to a v1 build_request?  You can't run the operators metadata plugin in any of the arrangements supported by v1 build_request.

Reply:
Removed